### PR TITLE
style: GEOT-104: four hundred four component styleguide

### PR DESCRIPTION
**Pull request by ra8ga**

### CODE REVIEW

**Comment by ra8ga** [comment]
> animation is broken, does not end at the bottom of the page, there is around 200px gap

fixed now

File: packages/ui/FourHundredFour/styles.scss [hunk 1]
```diff
@@ -1,37 +1,112 @@
...
+  max-width: 100%;
+  width: 100%;
+  height: 100%;
+  background: repeating-linear-gradient(
+    180deg,
+    rgba(0, 0, 0, 0) 0,
+    rgba(0, 0, 0, 0.3) 50%,
+    rgba(0, 0, 0, 0) 100%
+  );
+  background-size: auto 4px;
```
Comment (mariusz-kabala) [inline packages/ui/FourHundredFour/styles.scss]: why 4px?

File: packages/ui/FourHundredFour/styles.scss [hunk 2]
```diff
@@ -1,37 +1,112 @@
...
+  z-index: 99;
```
Comment (mariusz-kabala) [inline packages/ui/FourHundredFour/styles.scss]: why z-index is here

File: packages/ui/FourHundredFour/styles.scss (hunk 1 above)
Comment (ra8ga) [reply packages/ui/FourHundredFour/styles.scss]: to sa linie ktore tworza efekt "ekranu" - 5px tez bedzie ok

Comment (ra8ga) [reply packages/ui/FourHundredFour/styles.scss]: poprawione

File: packages/ui/FourHundredFour/styles.scss (hunk 2 above)
Comment (ra8ga) [reply packages/ui/FourHundredFour/styles.scss]: removed

**Pull request merged**